tests/client/test_redirects.py

```diff
@@ -348,19 +380,19 @@ async def test_can_stream_if_no_redirect():
 
 @pytest.mark.usefixtures("async_environment")
 async def test_cannot_redirect_streaming_body():
-    client = AsyncClient(transport=MockTransport())
+    client = AsyncClient(transport=AsyncMockTransport())
     url = "https://example.org/redirect_body"
 
     async def streaming_body():
-        yield b"Example request body"
+        yield b"Example request body"  # pragma: nocover
```

Comment:
Had to add this since we need a streaming body for this test, although is never consumed.